test/test_tensorclass_nofuture.py

```diff
@@ -718,6 +943,7 @@ class MyClass:
     assert (tc_dest == tc).all()
     assert tc_dest.y.batch_size == tc.y.batch_size
     assert isinstance(tc_dest.y.x, MemmapTensor)
+    assert tc_dest.z == z
 
 
```

Comment:
I don't see tests for get_attr

Reply:
yep, exactly because you use the get_attr in the set_attr and, implicitly, assume it works correctly, which maybe not the case.

Reply:
Added test case for getattr